src/main/scala/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.scala

```diff
@@ -386,13 +386,13 @@ class CollectDuplexSeqMetrics
       case _         => unreachable(s"Found ${ssGroups.size} single strand families in a double strand family!")
     }
 
-    val umi1s = new ArrayBuffer[String] // ab UMI 1 (and ba UMI 2) sequences
-    val umi2s = new ArrayBuffer[String] // ab UMI 2 (and ba UMI 1) sequences
+    val umi1s = Seq.newBuilder[String] // ab UMI 1 (and ba UMI 2) sequences
+    val umi2s = Seq.newBuilder[String] // ab UMI 2 (and ba UMI 1) sequences
```

Comment:
Arrays are fast, is this fast?

Reply:
Ok, so a little poking, and in 2.12 and 2.13, `Seq.newBuilder` yields a `ListBuffer`, and `IndexedSeq.newBuilder` yields a `VectorBuilder` which uses arrays under the covers and is fairly smart about not re-allocating stuff to make the immutable collection.  I'll change over to `IndexedSeq.newBuilder` where I had `Seq.newBuilder.`